.gitignore

```
@@ -1,11 +1,14 @@
 .DS_Store
+*.key
 config.yaml
 config.yml
 dev-config.yaml
 !.circleci/config.yml
 **kubeconfig**
 certs
 /secrets
+test-*
+*__pycache__*
```

```suggestion
__pycache__
```

AFAIK you can just have `__pycache__` here. Was there another use case than matching a whole dir name?